src/layouts/components/Header.tsx

```diff
@@ -227,10 +227,11 @@ function HeaderNetworkMenu (): JSX.Element {
   )
 }
 
-function HeaderLink (props: { text: string, pathname: string, className: string }): JSX.Element {
+function HeaderLink (props: { text: string, pathname: string, className: string}): JSX.Element {
```

Comment:
```suggestion
function HeaderLink (props: { text: string, pathname: string, className: string }): JSX.Element {
```

should revert changes like this, I was wondering what changed